content/blog/2023-02-28-ethereum-classic-is-a-commodity-ethereum-is-a-security/index.md

```diff
@@ -0,0 +1,140 @@
+
+![The crowdsale.](./2.png)
+
+From 2014 to 2016 both Ethereum and Ethereum Classic were one project, so we refer to them in this period as ETH/ETC. 
+
+The project was financed with a crowdsale. Based on the Howey test it may be inferred that the ETH/ETC [crowdsale was, indeed, a security](https://twitter.com/WendellMoone/status/1623760982672564226). 
+
+It was [founded](https://en.wikipedia.org/wiki/Ethereum) by Vitalik Buterin; then he partnered with several individuals to form a group of co-founders; they promised the expectation of profits out of the investment; created a foundation with the name of the network, even registering its trademarks; sold a unit of value previous to the creation of the cryptocurrency to gather funds for development; and promoted the crowdsale and Ethereum’s features, benefits, and appreciation potential just like any initial public offering of a stock would have been promoted.
+
+However, this status as a security was likely limited to the period between the crowdsale and the launch of the network on July 30th of 2015. 
+
+![ETH/ETC launch.](./3.png)
+
+On July 30th of 2015, the initial securities of the project were transformed into the ETH cryptocurrency, which was subsequently categorized as a commodity by the CFTC.
+
+This step may be interpreted to be an exchange and exit of the initial common enterprise between investors, Vitalik Buterin, his partners, and the Ethereum Foundation as the director of the project.
+
+From here, the project became a truly decentralized, proof of work based, public blockchain. 
+
+![The Ethereum split.](./7.png)
+
+Even when Ethereum split from the ETC mainnet in 2016 due to the DAO hard fork, ETH kept its decentralization because it remained as a proof of work blockchain.
+
+This period lasted until September 15th of 2022.
+
+![The ETH migration to Proof of Stake.](./8.png)
+
+Starting in September 15th of 2022, Ethereum cannot be described as a decentralized project because it migrated to Proof of Stake. Indeed, the moment it migrated, 51% of the blocks were censored by mega-staking pools to comply with international sanctions imposed by the U.S. Office of Foreign Assets Control (OFAC). And, a few months later up to 70% of its blocks were censored.
+
+As said before, PoS depends on large and entrenched pool operators that control the network. This, together with the control and influence that the Ethereum Foundation had from before, and the fact that the developers have incredible clout in the decisions about the protocol (for example, changing the supply of the token 6 times in its history) make it a centralized project dependent on others from the units’ of value perspective.
+
+![BTC remains a commodity and ETC as well.](./6.png)
+
+In the meantime, Bitcoin has always maintained its status as a commodity as determined by the CFTC in 2015, and ETC is functionally a commodity, just like BTC, because it has exactly the same consensus design with its decentralist guarantees.
```

Comment:
Line 114:  Remove run-on sentence
```suggestion
In the meantime, Bitcoin has always maintained its status as a commodity as determined by the CFTC in 2015. ETC is functionally a commodity, just like BTC, because it has exactly the same consensus design with its decentralist guarantees.
```